src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/InstalledAndTransitivePackageFeed.cs

```diff
@@ -28,12 +28,17 @@ public override async Task<SearchResult<IPackageSearchMetadata>> ContinueSearchA
             var searchToken = continuationToken as FeedSearchContinuationToken ?? throw new InvalidOperationException(Strings.Exception_InvalidContinuationToken);
             cancellationToken.ThrowIfCancellationRequested();
 
+            PackageCollectionItem[] matchesInstalled = PerformLookup(_installedPackages.GetLatestCore(), searchToken);
```

Comment:
No need to cache this in a variable since it's only ever used once.

Reply:
Removed 'caching' variables.